doc/setup/install.rst

```diff
@@ -106,7 +106,7 @@ Linux
 Debian/Ubuntu
 '''''''''''''
 
-Odoo 11.0 'deb' package currently supports `Debian Stretch`_, `Ubuntu Xenial`_,
+Odoo 12.0 'deb' package currently supports `Debian Stretch`_, `Ubuntu Xenial`_ (18.04),
```

Comment:
ubuntu xenial is the 16.04 ([history](https://en.wikipedia.org/wiki/Ubuntu#Releases)). As almost nobody remembers the name of the release, maybe we could remove the 3 versions and just replace by "supports Debian Stretch and  Ubuntu 18.04 or above"

Reply:
hahaha ok

Reply:
done the change :)